src/mmw/apps/modeling/mapshed/calcs.py

```diff
@@ -96,11 +94,11 @@ def erosion_coeff(ws, season):
     `rain_cool` instead.
     """
 
-    avg_warm = np.mean([w.rain_warm for w in ws])
-    avg_cool = np.mean([w.rain_cool for w in ws])
+    avg_warm = float(sum([w.rain_warm for w in ws])) / len(ws)
+    avg_cool = float(sum([w.rain_cool for w in ws])) / len(ws)
```

Comment:
That should be fine. The values of `rain_warm` and `rain_cool` are limited to 3 digits of precision.

```
mmw=# SELECT rain_warm, rain_cool FROM ms_weather_station LIMIT 10;
 rain_warm | rain_cool 
-----------+-----------
     0.340 |     0.150
     0.340 |     0.150
     0.190 |     0.150
     0.190 |     0.150
     0.270 |     0.210
     0.280 |     0.110
     0.228 |     0.118
     0.228 |     0.118
     0.217 |     0.112
     0.238 |     0.124
(10 rows)
```